update decision field descriptions in traffic events reference tables (#875)
ngrok/ngrok-docs: docs/obs/reference.mdx

Triggers when an HTTP request completes.

| Field                         | Type                                  | Description                                                                                                                                                                                                                                      |
| ----------------------------- | ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |

> | backend.connection_reused     | bool                                  | True if ngrok reused a TCP connection to transmit the HTTP request to the upstream service.                                                                                                                                                      |

Reviewer: ```suggestion
| backend.connection_reused     | bool                                  | Will be `true` if ngrok reused a TCP connection to transmit the HTTP request to the upstream service.                                                                                                                                                      |
```

> | basic_auth.decision           | string                                | ‘allow’ if the Basic Auth module permitted the request to the upstream service, ‘block’ if not permitted, and ’invalid’ if the module did not run.                                                                                               |

Reviewer: ```suggestion
| basic_auth.decision           | string                                | Will be ‘allow’ if the Basic Auth module permitted the request to the upstream service, ‘block’ if not permitted, and ’invalid’ if the module did not run.                                                                                               |
```

| http.response.body_length     | int64                                 | The size of the response body in bytes                                                                                                                                                                                                           |
| http.response.headers         | Map&lt;string, List&lt;string&gt;&gt; | A map of normalized response headers. Header keys are capitalized and header values are lowercased.                                                                                                                                              |
| http.response.status_code     | int32                                 | The status code of the response returned by the ngrok edge                                                                                                                                                                                       |

> | ip_policy.decision            | string                                | ‘allow’ if IP Policy module permitted the request to the upstream service, ‘block’ if not permitted, and ’invalid’ if the module did not run.                                                                                                    |

Reviewer: ```suggestion
| ip_policy.decision            | string                                | Will be ‘allow’ if IP Policy module permitted the request to the upstream service, ‘block’ if not permitted, and ’invalid’ if the module did not run.                                                                                                    |
```

| oauth.app_client_id           | string                                | The OAuth application client ID                                                                                                                                                                                                                  |

> | oauth.decision                | string                                | ‘allow’ if the OAuth module permitted the request to the upstream service, ‘block’ if not permitted, and ’invalid’ if the module did not run.                                                                                                    |

Reviewer: ```suggestion
| oauth.decision                | string                                | Will be ‘allow’ if the OAuth module permitted the request to the upstream service, ‘block’ if not permitted, and ’invalid’ if the module did not run.                                                                                                    |
```

| tls.client_cert.subject.cn    | string                                | The subject common name of the client’s leaf TLS certificate in the Mutual TLS handshake                                                                                                                                                         |
| tls.version                   | string                                | The version of the TLS protocol used between the client and the ngrok edge                                                                                                                                                                       |
| traffic_policy.logs           | List&lt;Map&lt;string, any&gt;&gt;    | The collection of logs generated by any configured [Traffic Policy Log](/http/traffic-policy/actions/log) actions                                                                                                                                |

> | webhook_verification.decision | string                                | ‘allow’ if the Webhook Verification module permitted the request to the upstream service, ‘block’ if not permitted, and ’invalid’ if the module did not run.                                                                                     |

Reviewer: ```suggestion
| webhook_verification.decision | string                                | Will be ‘allow’ if the Webhook Verification module permitted the request to the upstream service, ‘block’ if not permitted, and ’invalid’ if the module did not run.                                                                                     |
```

| conn.server_port    | int32                              | **Filterable.** The port that the connection for this request came in on                                                                                                                                                                                                                                                                                                                                                                                         |
| conn.start_ts       | timestamp                          | The timestamp when the TCP connection to the ngrok edge is established                                                                                                                                                                                                                                                                                                                                                                                           |

> | ip_policy.decision  | string                             | ‘allow’ if IP Policy module permitted the request to the upstream service, ‘block’ if not permitted, and ’invalid’ if the module did not run or, notably, if the closed TCP connection had been established for an endpoint configured as an HTTPS Edge. In this last case, the IP Policy module will run per http request and thus the decision field will only be populated on the corresponding [`http_request_complete.v0`](#http_request_completev0) event. |

Reviewer: ```suggestion
| ip_policy.decision  | string                             | Will be `allow` if the IP Policy module permitted the request to the upstream service, `block` if it denies the request, and `invalid` if the module didn’t run or if the TCP connection was established for an HTTPS Edge endpoint. In the latter case, the decision will only be available in the corresponding [`http_request_complete.v0`](#http_request_completev0) event. |
```

Reviewer: Tried to make it more direct and shorter
